app/lib/donations/controllers/DonorsChooseDonationController.js

```diff
@@ -252,12 +262,9 @@ DonorsChooseDonationController.prototype.postDonation = function(member, project
         try {
           logger.log('verbose', 'dc.requestToken POST user:%s body:%s', 
             donorPhone, body);
-          // Handles when we get 403/Forbidden but steps in when sufficient funds.
-          // if (typeof body !== 'object') {
-          //   self.endChatWithFail(member);
-          //   logger.error('dc.requestToken user:%s invalid body:%s', donorPhone, body);
-          //   return deferred.promise;
-          // }
+          if (typeof body !== 'object') {
+            logger.error('dc.requestToken user:%s invalid JSON:%s', donorPhone, body);
+          }
```

Comment:
Where this is strange is when we get the errors somewhere else (which is why I commented this block out in the first place). Testing again with prod credentials where API is up, but our account doesn't have enough $$: What's odd is that wouldn't expect our `'dc.requestToken` to log an error... but I think this is due to promise code/callbacks I haven't fully grasped yet.



> 1:55:46 PM web.1 |  debug: dc.submitDonation user:[removed] proposalId:2115609
1:55:46 PM web.1 |  debug: dc.requestToken POST https://apisecure.donorschoose.org/common/json_api.html?APIKey=[removed] user:[removed]
1:55:46 PM web.1 |  error: dc.requestToken user:[removed] invalid JSON:
1:55:46 PM web.1 |  {"statusCode":"0","statusDescription":"success","token":"[removed]"}
1:55:46 PM web.1 |  debug: dc.requestToken success user:[removed]
1:55:46 PM web.1 |  debug: dc.postDonation POST user:[removed] proposalId:2115609 email:aschachter@dosomething.org first:Aaron
1:55:47 PM web.1 |  error: dc.postDonation POST failed user:[removed] proposalId:2115609 body:{"statusCode":"1","statusDescription":"insufficient balance","donationId":0,"remainingProposalAmount":"-1","proposalURL":"https://admin.donorschoose.org/project/-/-1/?verify=0&amp;utm_source=api&amp;utm_medium=donate&amp;utm_content=bodylink&amp;utm_campaign=[removed]&amp;cacheBust=X%3e"}